test_unstructured/documents/test_html.py

```diff
@@ -329,15 +328,6 @@ def test_read_html_doc(tmp_path: pathlib.Path):
     ]
 
 
-# -- HTMLDocument.from_pages() -------------------------------------------------------------------
-
-
-def test_HTMLDocument_can_construct_from_existing_pages():
-    page = Page(number=0)
-    html_document = HTMLDocument.from_pages([page])
-    assert html_document.pages == [page]
```

Comment:
Why can this be removed?

Reply:
Constructing an `HTMLDocument` instance from `Page` objects was a legacy capability, probably doubtful even then because no HTML source code would then be in the object, and now only used by this single test. So this test and the alternate constructor that enabled it are no longer useful.